kotlin/custom-dictionary-style/README.metadata.json

```diff
@@ -13,6 +13,12 @@
         "Style",
         "Unique value"
     ],
+    "provision_from": [
+        "https://arcgisruntime.maps.arcgis.com/home/item.html?id=751138a2e0844e06853522d54103222a"
+    ],
+    "provision_to": [
+        "/Restaurant.stylx"
```

Comment:
just a question, does it matter that the data will be stored in two different ways? e.g. this way vs storing it in an ArcGIS/Samples directory. I wondered if the former is legacy from using SD card storage? And going forward with Kotlin we'd keep with the "/dataString.tpk" format?

Reply:
Eagle eyes! Yes, alot of the longer preamble in the paths ArcGIS/Samples/<data_type>/ will be dropped as we move away from SD card storage towards scoped storage